client/src/utils/prettifyStatus.js

```diff
@@ -19,6 +19,10 @@ export const prettifyStatus = (status, id, tabValue, handleEngage) => {
   const toolTip = isWithdrawn
     ? 'Participant is no longer available.'
     : 'This candidate was hired by another employer.';
+
+  const hideArchiveButton = ['Hired Candidates', 'Archived Candidates'].includes(tabValue)
```

Comment:
Since includes returns a boolean, it seems a bit redundant to have a ternary here.

Reply:
Good catch ..... will update